local-cli/generator-ios/templates/app/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle"];
+  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
Shouldn't this actually be `index.bundle?platform=ios`? Just like how you can write `require('button')` and the system automatically loads button.ios.js, it seems consistent for `index.bundle` to automatically refer to `index.ios.js`.